README.md

```diff
@@ -349,11 +349,16 @@ export interface TsdxOptions {
 #### Example: Adding Postcss
```

Comment:
I can't comment on the lines above this, but could you replace the typedef above with a relative link to the type file instead? That's what I meant when I said "Also replace the `TsdxOptions` snippet there to a link to the typing file"

Reply:
done